core/domain/html_cleaner_test.py

```diff
@@ -176,7 +177,7 @@ def test_get_rte_components(self):
             '</oppia-noninteractive-video><br></p>'
         )
 
-        expected_components = [
+        expected_components: List[Dict[str, Any]] = [
```

Comment:
As we can see here, the keys in those dictionaries are different and for that, it is hard to use `TypedDict` here. Hence, we are using `Any`.

Reply:
ditto as above